lumicks/pylake/force_calibration/tests/test_active_calibration.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import scipy.constants
+import pytest
+from lumicks.pylake.force_calibration.power_spectrum_calibration import (
+    calculate_power_spectrum,
+    fit_power_spectrum,
+)
+from lumicks.pylake.force_calibration.calibration_models import ActiveCalibrationModel
+from .data.simulate_calibration_data import generate_test_realisation_ideal
```

Comment:
`generate_test_realisation_ideal` is such a strange name. I guess `realisation` has a specific meaning I'm unaware of?

Reply:
maybe change it to `realization` 😉

Reply:
As Dean would say: usually the best place to find a name is in your documentation.

```py
    """Generate test data to test active calibration.
```
So maybe just `generate_active_calibration_test_data`.

Changing this is a bit out-of-scope for this PR so please feel free to move on and revisit it later if you feel is the best option.

Reply:
Sounds like a good name. I'll take it 👍